e2e-tests/src/password_filter.py

```diff
@@ -1,13 +0,0 @@
-import re
-import logging
-
-
-class PasswordFilter(logging.Filter):
-    """Filter class to obscure sensitive information from logging."""
```

Comment:
Did you remove this completely? I meant vkey can be public but not skey.

Reply:
Oh I see, you moved the log_filter to src.

Reply:
yes, we had two filters, I merged them and moved to src